src/utils/viewer/viewer.cpp

```diff
@@ -2,9 +2,11 @@
 // This source code is licensed under the MIT license found in the
 // LICENSE file in the root directory of this source tree.
 
+#include <Corrade/Utility/Resource.h>
 #include "esp/gfx/Viewer.h"
 
 int main(int argc, char** argv) {
+  CORRADE_RESOURCE_INITIALIZE(ShaderResources);
```

Comment:
This is only needed because the shader resources are put to an intermediate static library, if these would be put directly into the executable or the shared lib then the linker wouldn't throw those away as unreferenced.

But that's not possible here. In `Magnum::Shaders` I'm doing it in shader constructors, so here it would be like this:

```cpp
// outside of any namespace, ideally defined just once
static void importShaderResources() {
    CORRADE_RESOURCE_INITIALIZE(ShaderResources)
}

namespace esp { 
namespace gfx { 

GenericShader::GenericShader(...) {
    if(!Corrade::Utility::Resource::hasGroup("default-shaders"))
        importShaderResources();

    ...
}
```

and similarly for the PTex shader. Then you don't need this here and the Python part will work as well.